DuckDuckGo/NetworkProtection/AppTargets/BothAppTargets/NetworkProtectionTunnelController.swift

```diff
@@ -800,7 +806,7 @@ final class NetworkProtectionTunnelController: TunnelController, TunnelSessionPr
             Logger.networkProtection.error("TunnelController found no token")
             return nil
         }
-    }
+        }
```

Comment:
🧐

Reply:
oops, strange that lint fix didn't do anything here, fixed.